hsds/chunk_sn.py

```diff
@@ -87,35 +88,14 @@ async def get_slices(app, select, dset_json, bucket=None):
         raise HTTPBadRequest(reason=msg)
 
     dims = getShapeDims(datashape)  # throws 400 for HS_NULL dsets
-    maxdims = getDsetMaxDims(dset_json)
-
-    # refetch the dims if the dataset is extensible and request or hasn't
-    # provided an explicit region
-    if isExtensible(dims, maxdims) and (select is None or not select):
-        kwargs = {"bucket": bucket, "refresh": True}
-        dset_json = await getObjectJson(app, dset_id, **kwargs)
-        dims = getShapeDims(dset_json["shape"])
-
-    slices = None  # selection for read
```

Comment:
Should `slices` still be declared before the `try` block in the new version of this function?

Reply:
updated